Double-click-to-edit/DoubleClickToEdit.plugin.js

```diff
@@ -222,20 +262,6 @@ module.exports = class DoubleClickToEdit {
 		if (!instance)
 			return;
 
```

Comment:
```suggestion
		//When selecting text it might be useful to copy.
		const copyKeyHeld = this.checkForModifier(this.doubleClickToCopy, this.copyModifier, e);
		if (copyKeyHeld)
			this.copyToClipboard(document.getSelection().toString());
```
You'll want to copy the first moment you know it's a message before the page might be changed.